homeassistant/components/modbus/modbus.py

```diff
@@ -196,7 +196,7 @@ def __init__(self, hass, client_config):
         }
 
     @property
-    def name(self):
+    def name(self):  # pragma: no cover
```

Comment:
This property is only used in log messages it seems. Maybe we can remove it?

Reply:
I got a stack trace when running hass but not when running pytest. This can of course be due that class ModbusHub do not inherit from a HA class ?

If it is OK with you, I will keep it for now.

Reply:
I'm not following. What stack trace did you get?

Reply:
Let me try again, it was in my production environment, and I did not keep the log file. If I remember right it was a file in homeassistant/helpers that did not like it (or something else I did).

Reply:
Found it, some of the platform used the name, in case of a strange response, and if not present it platform breaks down (produces a stack trace).

Removed the use of _hub.name, and remove property name from ModbusHub.